lib/tasks/organisations.rake

```diff
@@ -8,6 +8,22 @@ namespace :organisations do
     end
   end
 
+  desc "Dry run for update organisations table using data from GOV.UK"
+  task fetch_dry_run: :environment do
+    include AdvisoryLock::DSL
+
+    ActiveRecord::Base.transaction do
+      puts("Beginning dry run organisation fetch")
+
+      with_lock("forms-admin:organisations:fetch") do
+        OrganisationsFetcher.new.call
+      end
```

Comment:
Even though this is a very small bit of logic, I reckon it's probably worth extracting into a method - just to make extra clear that this is a dry run testing the functionality of the `fetch` task, and if we update the `fetch` task this should be updated too.

Reply:
I've refactored it to abstract the task into a separate method - and the two tasks now share it. I've also added a bit of messaging output so that you can tell that something's happened, as before it was completely silent which I found a little unnerving.